modules/file-integrity-understanding-file-integrity-cr.adoc

```diff
@@ -23,10 +23,34 @@ metadata:
   name: worker-fileintegrity
   namespace: openshift-file-integrity
 spec:
-  nodeSelector:
+  nodeSelector: <1>
       node-role.kubernetes.io/worker: ""
-  config: {}
+  tolerations: <2>
+  - key: "myNode"
+    operator: "Exists"
+    effect: "NoSchedule"
+  config: <3>
+    name: "myconfig"
+    namespace: "openshift-file-integrity"
+    key: "config"
+    gracePeriod: 20 <4>
+    maxBackups: 5 <5>
+    initialDelay: 60 <6>
+  debug: false
+status:
+  phase: Active <7>
 ----
+<1> Defines the selector for scheduling node scans.
+<2> Specify `tolerations` to schedule on nodes with custom taints. When not specified, a default toleration allowing running on master and infra nodes is applied.
+<3> Define a `ConfigMap` containing an AIDE configuration to use.
+<4> The number of seconds to pause in between AIDE integrity checks. Frequent AIDE checks on a node may be resource intensive, so it might be useful to specify a longer interval. Default is 900 seconds (15 minutes).
+<5> The maximum number of AIDE database and log backups (leftover from the re-init process) to keep on a node. Older backups beyond this number are automatically pruned by the daemon. Default is set to 5.
+<6> The number of seconds to wait before starting the first AIDE integrity check. Default is set to 0.
```

Comment:
@sheriff-rh Could you also add spec.config.initialDelay to important-file-integrity-object-attributes part? Thanks. https://57230--docspreview.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-configuring.html#important-file-integrity-object-attributes_file-integrity-operator